types/amqp/index.d.ts

```diff
@@ -0,0 +1,425 @@
+// Type definitions for amqp 0.2
+// Project: https://github.com/postwait/node-amqp#readme
+// Definitions by: My Self <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import * as net from 'net'
+import * as events from 'events'
+
+declare namespace amqp {
```

Comment:
Instead of declaring a namespace and `export =`-ing it, just remove the namespace wrapper and directly export everything inside it.